### ISSUE TITLE
Improving the UI of the webpage.

### ISSUE THREAD

**Issue by YashMeh** [YashMeh/panda-portfolio#19]
## Description
The UI of the resume page can be improved, lets discuss what all we need in this thread @bhumijgupta @rampa2510

**Comment by bhumijgupta** [comment]
I would suggest creating some predefined templates and user can choose from any one of them

**Comment by rampa2510** [comment]
I agree with @bhumijgupta but then the templates would have to follow the same structure else we will have to create separate functions for individual templates to create the webpage.

**Comment by YashMeh** [comment]
Okay then, lets create some templates then and THEN take this thing forward.
